isotovideo

```diff
@@ -31,23 +31,67 @@ ANSI_COLORS_DISABLED or NO_COLOR can be set to any value to disable colors.
 
 Show the current program version and test API version
 
+=item B<-e, -?, --exit-status-from-test-results>
+
+isotovideo will exit with status 1 when a test module fails.
+
 =item B<-h, -?, --help>
 
 Show this help.
 
-=head1 TEST PARAMETER
+=back
+
+=head1 TEST PARAMETERS
 
 All additional command line arguments specified in the C<key=value> format are
 parsed as test parameters which take precedence over the settings in the
 vars.json file. Lower case key names are transformed into upper case
 automatically for convenience.
 
+=head1 EXIT CODES
+
+=over 4
+
+=item B<0 - SUCCESS>
+
+isotovideo successfully executed a complete run. In the case of C<--exit-status-from-test-results> flag additionally all test modules passed or softfailed.
+
+C<isotovideo> can return non-zero exit codes on test module failures with the
+C<--exit-status-from-test-results> flag.
```

Comment:
Mentioning this flag again seems a bit redundant:
```suggestion
```

Reply:
Addressed in [5eb5dfb8](https://github.com/os-autoinst/os-autoinst/pull/2419/commits/5eb5dfb82b6cd3bdfeda8794909d8931a71bf526).